src/sinks/datadog/logs/sink.rs

```diff
@@ -190,8 +192,10 @@ impl RequestBuilder {
             {
                 let log = event.as_mut_log();
                 log.rename_key_flat(self.log_schema_message_key, "message");
-                log.rename_key_flat(self.log_schema_timestamp_key, "date");
                 log.rename_key_flat(self.log_schema_host_key, "host");
+                if let Some(Value::Timestamp(ts)) = log.remove(self.log_schema_timestamp_key) {
+                    log.insert(TIMESTAMP_KEY, Value::Integer(ts.timestamp_millis()));
+                }
```

Comment:
Is doing this without the `*_flat` functions going to noticeably impact perf?

Reply:
don't think so, changed it 👍